app/assets/stylesheets/layouts/_results.scss

```diff
@@ -19,6 +19,33 @@
   }
 }
 
+.l-results__firm-name {
+  @include column(12);
+
+  margin-left: 0;
+  margin-right: 5%;
+
+  @include respond-to($mq-m) {
+    @include column(8);
+
+    margin-left: 0;
+    margin-right: 5%;
+  }
+}
+
+.l-results__profile-link {
+  @include column(12);
+
+  text-align: right;
+  margin-right: 0;
+
+  @include respond-to($mq-m) {
+    @include column(4);
+
+    margin-right: 0;
+  }
+}
+
 .l-results__heading {
   @include column(12);
 
```

Comment:
Sadly these are all over the MAS codebase and are the main reason blocking us upgrading SASS. Would be great if you can fix this one just so there's one less to deal with when we do get round to fixing the rest!